tests/e2e/specs/plugin-updater.test.js

```diff
@@ -20,13 +19,12 @@ describe( 'Jetpack updater', () => {
 	catchBeforeAll( async () => {
 		await prepareUpdaterTest();
 		const url = getNgrokSiteUrl();
-
-		await execMultipleWpCommands(
```

Comment:
Why did you split these up? 

FWIW, it was done intentionally, to speed up the wp-env execution time

Reply:
Debug purposes. I wanted to se what's happening after each command. I will change it back.